tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -90,19 +92,39 @@ def _process_cancelled_event(self, calendar_event):
             todoist_item = self.todoist.get_item_by_id(todoist_id)
             if todoist_item is not None:
                 self.todoist.delete_item(todoist_item)
+    
+    def _process_updated_event(self, calendar_event):
+        print('Updating event|', calendar_event)
+        todoist_item = self.todoist.get_item_by_id(_todoist_id(calendar_event))
+        if todoist_item is None:
+            print('Invalid event??')
+            return None
+        next_occurence = calendar_event.next_occurrence()
+        if next_occurence is None:
+            self.todoist.delete_item(todoist_item)
+            return None
+        todoist_item.set_due(next_occurence, calendar_event.recurrence_string())
```

Comment:
introduce a new function in `TodoistItem` that is 'move_due' which only takes a date.